### ISSUE TITLE
feature request: update the requests dependence

### ISSUE THREAD

**Issue by liudonghua123** [localstack/localstack#7078]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature description

Please update the requests dependence to support the latest 2.x version. Or I will get the following warning or errors.

```
localstack 1.2.0 requires requests<2.26,>=2.20.0, but you have requests 2.28.1 which is incompatible.
localstack-ext 1.2.0 requires requests<2.26,>=2.20.0, but you have requests 2.28.1 which is incompatible.
```

### 🧑‍💻 Implementation

update the requests dependence to something like `requests<3.0,>=2.20.0`

**Comment by lesteenman** [comment]
Running into this as well. It prevents me from updating my `docker` python dependency from 5.0 to 6.0+, because that package requires a requests version _higher than_ 2.26.

**Comment by dfangl** [comment]
Hi!
The pin is removed, and the change will be released in the localstack pypi package with 1.3.0.
If you want to use it immediatly, please upgrade localstack while allowing pre-releases, so `pip install --pre -U localstack`.
